desktop/apps/article/templates/amp_mixins.jade

```diff
@@ -61,6 +61,9 @@ mixin amp_video(section)
       amp-youtube(data-videoid=embed.getId(section.url) layout="responsive" width="480" height="270")
     else if section.url.match('vimeo')
       amp-vimeo(data-videoid=embed.getId(section.url) layout="responsive" width="480" height="270")
```

Comment:
I would audit whether there're other links that people post, otherwise not anyone I can think of.